app/views/static/buttons/generic_button_component.html.haml

```diff
@@ -0,0 +1,27 @@
+.clearfix
```

Comment:
Maybe let's not introduce *yet another* buttons partial, can you just use `generic_form_buttons` instead? Or do a separate PR that converts generic_form_buttons to a component first?